tests/unit/StaffNotation.spec.ts

```diff
@@ -66,4 +67,68 @@ describe('StaffNotation', () => {
     expect(wrapper.emitted('close').pop()).toEqual([false])
     wrapper.destroy()
   })
+
+  it('renders the modal correctly for Registrar\'s Notation', async () => {
```

Comment:
Thanks for adding these!

Instead of duplicating the tests for each filing type, do you want to try parameterizing the tests?

Example: https://github.com/bcgov/business-create-ui/blob/239d90d0fcd29649bbf16c195f0271dd3a5bf4aa/tests/unit/AgreementType.spec.ts#L64

Reply:
Thanks. I added the loop for the three staff filing types in my latest commit.

Reply:
Excellent, thanks!